### PR TITLE
ref: reduce instance waitfor timeout - transform it using helpers

### PR DESCRIPTION
**Pull request by boranx**
Use helpers function for polling mechanism
reduce timeout for _waitfor_ from 20 mins to 2 min

Execution Example: 
```shell
Using region: us-east-1
Created instance with ID: i-05085ec6471a11109
EC2 Instance: i-05085ec6471a11109 Running
Gathering and parsing console log output...
Terminating ec2 instance with id i-05085ec6471a11109
Summary:
All tests pass!
Success
```

### CODE REVIEW

File: pkg/cloudclient/aws/private.go [hunk 1]
```diff
@@ -237,39 +237,26 @@ func (c *Client) describeEC2Instances(ctx context.Context, instanceID string) (i
...
-			if descError, ok := descError.(awserr.Error); ok {
-				descErrorMsg = fmt.Sprintf("Could not get EC2 instance state: %s, terminating instance %s", descError.Code(), instanceID)
-			}
-
-			return fmt.Errorf("%s: %s", descError, descErrorMsg)
+			return false, descError // unhandled
 		}
-	}
 
-	c.logger.Info(ctx, "EC2 Instance: %s Running", instanceID)
```
Comment (fahlmant) [inline pkg/cloudclient/aws/private.go:271]: Why remove this log?

Comment (boranx) [reply pkg/cloudclient/aws/private.go:271]: we're not removing, but just _moving_: https://github.com/openshift/osd-network-verifier/pull/81/files/ab9f00239ccc70e9fa242c9d9aa907b89319a961#diff-0dfb65e2caeb4c025cfe6a19c3ed8e9add53107d57033e1be141011478911681R246

Comment (fahlmant) [reply pkg/cloudclient/aws/private.go:271]: ah missed that :)

**Review by fahlmant — APPROVED** [review]
/approve
/lgtm

**Pull request merged**